web/icard-web/App.js

```diff
@@ -1,8 +1,24 @@
 import { StatusBar } from "expo-status-bar";
 import { StyleSheet, Text, View } from "react-native";
-import Home from "./src/views/Home";
-import RegistrationView from "./src/views/RegistrationView";
+import AuthContext from "./src/context/AuthContext";
+import { useState } from "react";
+import Index from "./src/views/Index";
 
+
+import { GoogleOAuthProvider } from "@react-oauth/google";
+import { GoogleLogin } from "@react-oauth/google";
+
+import secret from "./secrets/client_secrets.json";
```

Comment:
I know I use the secrets as the way to get client ID. But can I ask you to turn these value into environment variable? You can use this library https://www.npmjs.com/package/react-native-dotenv